com.github.wwmm.PulseEffects.json

```diff
@@ -0,0 +1,267 @@
+{
+  "app-id": "com.github.wwmm.PulseEffects",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.24",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.gfortran-62"],
+  "command": "pulseeffects",
+  "rename-desktop-file": "pulseeffects.desktop",
+  "rename-icon": "pulseeffects",
+  "finish-args": [
+    "--socket=wayland",
+    "--socket=pulseaudio",
+    "--own-name=com.github.wwmm.pulseeffects",
+    "--persist=.config/PulseEffects",
```

Comment:
True. I suppose one way of seeing it is that the developer outsourced the GUI for profile selection to GtkFileChooseDialog. Except that is not entirely correct. One use case for the current implementation would be sharing profiles with other users over a network share.

But yes, that is a discussion for upstream. How should we continue though?